Tests/scripts/infrastructure_tests/release_notes_generator_test.py

```diff
@@ -293,6 +293,34 @@ def test_release_notes_summary_with_ignored_rns(self):
 
 
 class TestMergeVersionBlocks:
+    def test_spaced_content_entity_and_old_format(self):
+        """
+        Given
+        - Two release notes files, one with content entity instance wrapped with **, content entity type contains
+        spaces.
```

Comment:
Both are with `**` right?
```suggestion
        - Two release notes files with content entity instance wrapped with ** and entity type contains spaces.
```